lib/membrane_rtc_engine/endpoints/webrtc/track_sender.ex

```diff
@@ -35,7 +35,7 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.TrackSender do
 
   @impl true
   def handle_init(%__MODULE__{track: track}) do
-    {:ok, %{track: track, trackers: %{}}}
+    {:ok, %{track: track, trackers: %{}, keyframe_req_queue: MapSet.new()}}
```

Comment:
I wouldn't call it a "queue" if it's a set - "queue" suggest an ordered collection.
I'd suggest something like
```suggestion
    {:ok, %{track: track, trackers: %{}, requested_keyframes: MapSet.new()}}
```
or (bit too long)
```suggestion
    {:ok, %{track: track, trackers: %{}, pending_keyframe_requests: MapSet.new()}}
```
or
```suggestion
    {:ok, %{track: track, trackers: %{}, pending_keyframes: MapSet.new()}}
```